varats/varats/projects/perf_tests/feature_perf_cs_collection.py

```diff
@@ -56,14 +56,21 @@ class FeaturePerfCSCollection(VProject):
             ),
             Command(
                 SourceRoot("FeaturePerfCSCollection") /
-                RSBinary("SimpleFeatureInteraction"), "--enc", "--compress"
+                RSBinary("SimpleFeatureInteraction"),
+                "--enc",
+                "--compress",
+                label="SFI-enc-compress"
             )
         ],
         WorkloadSet(WorkloadCategory.MEDIUM): [
             Command(
                 SourceRoot("FeaturePerfCSCollection") /
-                RSBinary("SimpleBusyLoop"), "--iterations", str(10**7),
-                "--count_to", str(5 * 10**3)
+                RSBinary("SimpleBusyLoop"),
+                "--iterations",
+                str(10**7),
+                "--count_to",
+                str(5 * 10**3),
+                label="SBL-iterations-count-to"
```

Comment:
Thanks for catching this.

```suggestion
                label="SBL-iterations-10M-count-to-5K"
```
SimpleBusyLoop only has these two configuration options and both are required. They directly influence how many feature tracing events SBL will generate together with their frequency. I therefore recommend that we add this information to the label.